src/sentry/monitors/serializers.py

```diff
@@ -48,9 +48,13 @@ def get_attrs(self, item_list, user, **kwargs):
         environment_data = {}
         if self.environments:
             monitor_environments = defaultdict(list)
-            for monitor_environment in MonitorEnvironment.objects.filter(
-                monitor__in=item_list, environment__in=self.environments
-            ).select_related("environment"):
+            for monitor_environment in (
+                MonitorEnvironment.objects.filter(
+                    monitor__in=item_list, environment__in=self.environments
+                )
+                .select_related("environment")
+                .order_by("-last_checkin")
```

Comment:
does this make sense for multiple environments? or should I try to mirror the ordering with monitors?